### PR TITLE
Bulk validation fixes ms.prod

### PR DESCRIPTION
**Pull request by rflynn27**

### CODE REVIEW

**Comment by rflynn27** [comment]
Global effort to fix build validation errors

@Lauragra 

It is still incorrectly throwing column-header suggestions after making the necessary changes. Thanks!

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: api-reference/beta/resources/diagnostic.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "diagnostic resource type"
 description: "Information about an error or warning for a OneNote operation."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: onenote
```
Comment (Lauragra) [inline api-reference/beta/resources/diagnostic.md:6]: Change this value to "notes".

File: api-reference/beta/resources/externallink.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: "externalLink resource type"
 description: "A url that opens a OneNote page or notebook."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: onenote
```
Comment (Lauragra) [inline api-reference/beta/resources/externallink.md:6]: notes

File: api-reference/beta/resources/inferenceclassification.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: "inferenceClassification resource type"
 description: "Classification of a user's messages to enable focus on those that are more relevant or important to the user. "
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: outlook
```
Comment (Lauragra) [inline api-reference/beta/resources/inferenceclassification.md:6]: Change this to "mail".

File: api-reference/beta/resources/inferenceclassificationoverride.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: "inferenceClassificationOverride resource type"
 description: "Represents a user's override for how incoming messages from a specific sender should always be classified as"
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: outlook
```
Comment (Lauragra) [inline api-reference/beta/resources/inferenceclassificationoverride.md:6]: mail

**Pull request merged**